### PR TITLE
1742 persistant export image tooltip

### LINKED ISSUE #1742: persistent export image tooltip
This only happens after 100% resolution is clicked.

Screen Shot 2021-12-07 at 1 49 04 PM

### PR DESCRIPTION
**Pull request by Michaelazzz**
closes #1742

### CODE REVIEW

File: src/stores/AppStore.ts [hunk 1]
```diff
@@ -2227,21 +2227,24 @@ export class AppStore {
...
-                        // Trim filename before timestamp to 200 characters to prevent browser errors
-                        link.download = `${joinedNames}-image`.substring(0, 200) + `-${getTimestamp()}.png`;
-                        link.href = URL.createObjectURL(blob);
-                        link.dispatchEvent(new MouseEvent("click"));
-                    }, "image/png");
-                }
-                this.setIsExportingImage(false);
-            });
+            this.waitForImageData()
+                .then(() => {
```
Comment (veggiesaurus) [inline src/stores/AppStore.ts:2231]: might be simpler if we just switch this to an `async` function and use `await`

**Comment by kswang1029** [comment]
@Michaelazzz it seems the tooltip is still persistent. This is with Chrome v101

https://user-images.githubusercontent.com/20819712/168520273-d056d282-8b8d-4e6b-ace8-faf68511673e.mov

**Comment by kswang1029** [comment]
#1742 will be fixed via #1898

**Comment by kswang1029** [comment]
Fixed via #1898

**Pull request closed without merging**